CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ FetchContent_Declare(
     GIT_REPOSITORY https://github.com/PennyLaneAI/pennylane-lightning.git
     GIT_TAG      master 
 )
+set(ENABLE_KOKKOS OFF CACHE BOOL "Disable Kokkos")
```

Comment:
Shouldn't this docstring be "Enable Kokkos"?

Reply:
Nice catch! Fixed.